src/test/java/com/aws/greengrass/clientdevices/auth/api/UseCasesTest.java

```diff
@@ -102,33 +99,12 @@ void GIVEN_aUseCaseWithDependencies_WHEN_ran_THEN_itExecutesWithNoExceptions() {
     @Test
     void GIVEN_aUseCaseWithExceptions_WHEN_ran_THEN_itThrowsAnException() {
         UseCaseWithExceptions useCase = useCases.get(UseCaseWithExceptions.class);
-        assertThrows(InvalidConfigurationException.class, () -> { useCase.apply(null); });
+        assertThrows(UseCaseException.class, () -> { useCase.apply(null); });
     }
 
     @Test
     void GIVEN_aUseCaseWithParameters_WHEN_ran_itAcceptsTheParamsAndReturnsThem() {
         UseCaseWithParameters useCase = useCases.get(UseCaseWithParameters.class);
         assertEquals(useCase.apply("hello"), "hello");
     }
-
```

Comment:
Removed this since we are not providing the configuration like this anymore. We determined it would do more harm than good if/when we introduced threading to process use cases.